packages/cadl-ranch-specs/http/client/structure/default/mockapi.ts

```diff
@@ -0,0 +1,24 @@
+import { passOnSuccess, mockapi } from "@azure-tools/cadl-ranch-api";
+import { ScenarioMockApi } from "@azure-tools/cadl-ranch-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+Scenarios.Client_Structure_Service = passOnSuccess([
+  mockapi.patch("/one", (req) => {
```

Comment:
how to make sure the tests are all covered if they all share the same endpoints ?

Reply:
the full url for all of those is 
```
http://localhost:3000/client/structure/{client}/...
```

so make sure each endpoint here is `/client/structure/{client}/...` and replace `{client}` with the scenario name

Reply:
But the summary is saying the otherwise, should I change anything in generating the summary ?

Reply:
Yeah summary doesn’t take the server url into account. We should fix that

Reply:
Got it, I will see how to do that? maybe we should add some validation checks here to make sure the mockapi.ts is reflecting the real apis?

Reply:
Because it should have reported errors now, but it's not.

Reply:
Yeah but it it’s a little tricky when we have some path parameter to make sure things match and not find false positive.

Reply:
will track with #385